packages/docs/page-config/ui-elements/data-table/examples/Sorting.vue

```diff
@@ -30,25 +27,19 @@
   />
 
   <va-alert
```

Comment:
I think it makes demo harder to read and not that important. Let's have a demo in VueBook instead.